### PR TITLE
Pass SolutionServices in more places

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

File: src/Workspaces/Remote/ServiceHub/Services/BrokeredServiceBase.cs [hunk 1]
```diff
@@ -76,8 +76,8 @@ public virtual void Dispose()
         public RemoteWorkspace GetWorkspace()
             => WorkspaceManager.GetWorkspace();
 
-        public HostWorkspaceServices GetWorkspaceServices()
-            => GetWorkspace().Services;
+        public SolutionServices GetWorkspaceServices()
```
Comment (Youssef1313) [inline src/Workspaces/Remote/ServiceHub/Services/BrokeredServiceBase.cs:79]: > GetWorkspaceServices

Consider renaming to `GetSolutionServices`?

**Comment by CyrusNajmabadi** [comment]
@dibarbet ptal.

File: src/Workspaces/Remote/ServiceHub/Services/BrokeredServiceBase.cs (hunk 1 above)
Comment (CyrusNajmabadi) [reply src/Workspaces/Remote/ServiceHub/Services/BrokeredServiceBase.cs:79]: I'm primarily concerned with using the right types and not leaking out stuff that isn't necessary.  I'd prefer to keep these PRs minimal and not clutter with naming that doesn't substantially improve readability to me.  thanks!

**Review by dibarbet — APPROVED** [review]

**Comment by dibarbet** [comment]
@CyrusNajmabadi all the tests are crashing with a stackoverflow, dumps should be attached to the test runs
image

**Pull request merged**